src/test/java/teammates/test/cases/browsertests/InstructorCourseDetailsPageUiTest.java

```diff
@@ -239,14 +237,14 @@ private InstructorCourseDetailsPage getCourseDetailsPage() {
         return loginAdminToPage(detailsPageUrl, InstructorCourseDetailsPage.class);
     }
 
-    private boolean didStudentReceiveReminder(String courseName, String courseId, String studentEmail,
-                                              String studentPassword)
+    // TODO replace with GAE LocalMailService stub as we are checking the logic here, not reliability of email delivery
```

Comment:
Is there a way to make `RetryManager` start linearly first and after some time, then it starts to slow down significantly. 

For example, I would like to retry once per second until 1 minute before slowing down. Should I just manually retry until 1 minute then use RetryManager or something then?